src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedIfDepthCheck.java

```diff
@@ -50,6 +66,32 @@
  *   &lt;property name=&quot;max&quot; value=&quot;3&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Valid Code Example:</p>
+ * <pre>
+ * if (true) {
+ *    if (true) {
+ *       if (true) {
+ *          if (true) {} // OK
+ *          else {}
+ *       }
+ *    }
+ * }
+ * </pre>
+ * <p>Invalid Code Example:</p>
+ * <pre>
+ * if (true) {
+ *    if (true) {
+ *       if (true) {
+ *          if (true) {
+ *             if (true) { // Nested if-else depth is 5 (max allowed is 3).
+ *                if (true) {}
```

Comment:
here depth is 5. (violation)
put violations on both the line.

Reply:
done.

Reply:
resolved